src/screens/ArticlePage/index.js

```diff
@@ -0,0 +1,121 @@
+import React, {useState, useEffect} from 'react';
+import { View, Text, Button, StyleSheet, Dimensions, ScrollView, Image, TouchableOpacity} from 'react-native';
+import appApiClient from '../../api/appApiClient';
+const { width, height } = Dimensions.get('window')
+
+
+
+export default function ArticlePage({route , navigation}) {
+  console.log(route.params.id)
+  const [uniqueArticle, setUniqueArticle] = useState({});
+useEffect(() => {
+  findOneArticle()
```

Comment:
do u use prettier ? and did u check yarn lint for errors in your files (do not pay attention other errors which  we are aware of)

Reply:
Yes, I ran yarn lint and there were no warnings in my files